src/pages/Checkout/CheckoutPage.vue

```diff
@@ -354,11 +356,24 @@ export default {
 			})
 				.then(() => {
 					return Promise.all([
-						client.query({ query: initializeCheckout, fetchPolicy: 'network-only' })
+						client.query({ query: initializeCheckout, fetchPolicy: 'network-only' }),
+						client.query({ query: experimentQuery, variables: { id: 'upsells_checkout' } }),
 					]);
 				});
 		},
 		result({ data }) {
+			const upsellsExperiment = this.apollo.readFragment({
```

Comment:
Please move all of this new code from the `result` block to either `created` or `mounted`